src/clarity-angular/wizard/providers/wizard-navigation.ts

```diff
@@ -58,11 +58,10 @@ export class WizardNavigationService implements OnDestroy {
             } else {
                 this.cancel();
             }
-            // SPECME
         });
 
         this.pagesResetSubscription = this.pageCollection.pagesReset.subscribe(() => {
-            this.setLastEnabledPageCurrent();
+            this.setFirstPageCurrent();
```

Comment:
Ah. No. This is just being more explicit about what we want the wizard to do here. Reset should reset the navigation to its pristine state. 

This change here might actually fix issue #810